### PR TITLE
Version 1.1, fix some bugs.

### PR DESCRIPTION
**Pull request by lhy101**

### CODE REVIEW

File: chatRoom.py [hunk 1]
```diff
@@ -19,20 +19,29 @@ def root():
...
     userPassword = request.form["userPassword"]
     checkState = userTab.login(userID, userPassword)
 
     if checkState == True: 
-        global curUserID
-        global curUserName
         curUserID = userID
         curUserName = userTab.getUser(userID)['name']
         print(curUserID)
+
```
Comment (soxziw) [inline chatRoom.py:37]: 速速去掉！

**Review by soxziw — DISMISSED** [review]
HURRY!!!
:P

File: chatRoom.py (hunk 1 above)
Comment (lhy101) [reply chatRoom.py:37]: No way~~~

**Review by soxziw — APPROVED** [review]
Nice job!

**Pull request merged**